src/mpcCoreKit.ts

```diff
@@ -617,7 +619,7 @@ export class Web3AuthMPCCoreKit implements ICoreKit {
     return this.atomicSync(async () => {
       await this.copyOrCreateShare(shareType, factorPub);
       await this.backupMetadataShare(factorKey);
-      await this.addFactorDescription(factorKey, shareDescription, additionalMetadata);
+      await this.addFactorDescription(factorKey, shareDescription, additionalMetadata, false);
```

Comment:
It would be safer to call it in serial. 
also most of the await is related to syncMetadata, since in atomicSync we already skip the syncMetadata call call await does not save too much time

Reply:
update addFactorDescription